lib/omniauth/strategies/twitter_ads.rb

```diff
@@ -2,8 +2,8 @@
 
 module OmniAuth
   module Strategies
-    class Twitter < OmniAuth::Strategies::Twitter
-      option :name, 'twitter'
+    class TwitterAds < OmniAuth::Strategies::Twitter
+      option :name, 'twitter_ads'
```

Comment:
@bobintornado if I remember correctly - if you change this value `twitter_ads`, that means everything that's previously under `twitter` as provider will need to change as well. meaning provider will now becomes `twitter_ads` instead of `twitter`. that requires a lot of updates - can we just stick to using `twitter`?